deployment/ansible/roles/nyc-trees.python/tasks/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+- name: Install Python Docutils
+  apt: pkg=python-docutils=0.11-3 state=present
```

Comment:
I feel like this should just be placed into the `azavea.beaver` role, or perhaps the `nyc-trees.beaver` role.

After all, `python-docutils` is not something required for installing Python, just Beaver right?